test/integration/targets/file/tasks/main.yml

```diff
@@ -62,6 +62,22 @@
       - "file4_result.changed == true"
       - "file4_result.mode == '0600'"
 
+- name: change file attribute "A"
+  file: path={{output_dir}}/baz.txt attributes=A
+  register: file_attributes_result
+  ignore_errors: True
+
+- name: reapply file attribute "A"
+  file: path={{output_dir}}/baz.txt attributes=A
+  register: file_attributes_result_2
+  when: file_attributes_result|changed
+
+- name: verify that the file was not marked as changed
+  assert:
+    that:
+      - "file_attributes_result_2.changed == false"
```

Comment:
Use test instead of filter, replace both `.changed` and `|changed` with `is changed` ([see](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guide_2.5.rst#jinja-tests-used-as-filters)).

Reply:
done